packages/preset-react-app/docs/getting-started/1.md

```diff
@@ -11,9 +11,11 @@ These lines will install Phenomic and its [react app preset](https://github.com/
 
 ⚠️ _If you don't have a ``package.json`` you can do this simple command to get
  a valid one:_
- 
+
 ```console
 echo '{"private":true}' > package.json
+# or you can also do
+yarn init
```

Comment:
Maybe yes. I think people know npm/yarn init anyway :)
But do not hesitate to make some others PRs if you feel the need, I like having discussions on docs!